openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KStream.java

```diff
@@ -144,7 +144,6 @@ public KStream startBroadcast(final StreamProcessor processor, final StreamDesc
 			}
 		});
 		outgoingMedia.addMediaFlowInStateChangeListener(evt -> log.warn("Media FlowIn :: {}", evt));
-		processor.addStream(this);
```

Comment:
We don't need to add stream here anymore, KStream is added in the constructor. So won't need to add it later on again.